helm-chart/renku/values.yaml

```diff
@@ -741,6 +741,7 @@ ui:
       image:
         repository: renku/renku-ui
         tag: "0.11.16"
+    supportLegacySessions: true
```

Comment:
Why is this a parameter on the UI only? Sessions may still be started by calling the API if the value is not consumed on the API.

Reply:
My understanding is that the Product doesn't think it's worth the effort to update the backend services, disabling the UI is all that's planned. I don't have a _solid_ reference, only https://www.notion.so/renku/Disable-Legacy-sessions-21e0df2efafc80fc920aed586557b74c?source=copy_link#22d0df2efafc80cc86b2fe82db531aab